consts/user.js

```diff
@@ -0,0 +1,18 @@
+const allowedUserFieldsForUpdate = {
+  photo: true,
+  firstName: true,
+  lastName: true,
+  address: {
+    country: true,
+    city: true
+  },
+  experience: true,
+  subject: true,
+  averageRating: true,
+  nativeLanguage: true,
+  appLanguage: true
+}
+
+module.exports = {
```

Comment:
also I pretty much believe that this filter should be inside the 'validation' folder, if anything.